APIdaze.SDK/Recordings/Recordings.cs

```diff
@@ -32,10 +34,7 @@ public async Task DownloadRecodingToFileAsync(string sourceFileName, string targ
             var restRequest = DownloadRequest(sourceFileName);
             var response = await Client.ExecuteTaskAsync(restRequest);
 
-            if (response.StatusCode != HttpStatusCode.OK)
-            {
-                throw new InvalidOperationException(response.ErrorMessage);
-            }
+            if (response.StatusCode != HttpStatusCode.OK) throw new InvalidOperationException(response.ErrorMessage);
 
             SaveFileToFolder(sourceFileName, target, response);
```

Comment:
Let's return the FileInfo of the created file as well for the sake of consistency between this and `DownloadRecordingToFile`.